server_modules/scores.js

```diff
@@ -15,6 +16,29 @@ function reduceToMap(key) {
     }
 }
 
+function changeScores(callback) {
```

Comment:
This now reads as if this is a typical 'Node-style' function that accepts a callback that's called when the operation is completed.

Please change name of `callback` to something that reflects its purpose, and add a doc comment to describe the function.